astrowidgets/core.py

```diff
@@ -426,6 +427,8 @@ def marker(self, val):
         marker_type = val.pop('type')
         if marker_type == 'circle':
             self._marker = functools.partial(self.dc.Circle, **val)
+        elif marker_type == 'point':
```

Comment:
This would become two marker types, `plus` and `cross`

Reply:
Hi I have made the changes as you requested! If there is anything wrong please let me know.